use-cases/xtclim/src/anomaly.py

```diff
@@ -1,114 +1,182 @@
+            print('Train average loss:', train_avg_losses)
+            print('Test average loss:', test_avg_losses)
+
+            self.log(train_avg_losses,
+                    'Average train loss',
+                    kind='metric')
+            self.log(test_avg_losses,
+                     'Average test loss',
+                     kind='metric')
+
+            # Clean-up
+            if self.strategy.is_main_worker and self.logger:
+                self.logger.destroy_logger_context()
+
+            self.strategy.clean_up()
+
+        else:
+            season = self.seasons
+            # load previously trained model
+            cvae_model = NNmodel.ConvVAE().to(self.device)
+            cvae_model.load_state_dict(self.model['model_state_dict'], strict=False)
+
+            distribute_kwargs = {}
+            # Distributed model, optimizer, and scheduler
+            cvae_model, _, _ = self.strategy.distributed(
+                cvae_model, **distribute_kwargs
+            )
+
+            if self.strategy.is_main_worker and self.logger:
+                 self.logger.create_logger_context()
+
+            for scenario in ['585', '245']:
+
+                # projection set and data loader
+                proj_time = pd.read_csv(f"input/dates_proj_{season}data_1memb.csv")
+                proj_data = np.load(f"input/preprocessed_1d_proj{scenario}_{season}data_1memb.npy")
+                n_proj = len(proj_data)
+                projset = [ ( torch.from_numpy(np.reshape(proj_data[i], (3, 32, 32))),
+                        proj_time['0'][i] ) for i in range(n_proj) ]
+                projloader = self.create_dataloaders(
+                     projset, batch_size=1, shuffle=False
+                )
+
+                # get the losses for each data set
+                # on various experiments to have representative statistics
+                proj_avg_losses, _, tot_proj_losses, _ = evaluate(cvae_model, projloader,
+                                                       projset, self.device, criterion,
+                                                       pixel_wise_criterion)
+
+                for i in range(1, n_avg):
+                    proj_avg_loss, _, proj_losses, _ = evaluate(cvae_model, projloader,
+                                                       projset, self.device, criterion,
+                                                       pixel_wise_criterion)
+                    tot_proj_losses = list(map(add, tot_proj_losses, proj_losses))
+                    proj_avg_losses += proj_avg_loss
+
+                tot_proj_losses = np.array(tot_proj_losses)/n_avg
+                proj_avg_losses = proj_avg_losses/n_avg
+
+                # save the losses time series
+                pd.DataFrame(tot_proj_losses).to_csv(f"outputs/proj{scenario}_losses_{season}1d_allssp.csv")
+                print(f'SSP{scenario} Projection average loss:', proj_avg_losses, 'for', season[:-1])
```

Comment:
Why do you use an f-string but then also use commas to separate statements? You should stick to one: either use "{}" for all, or commas for all.